tests/test_main.cpp

```diff
@@ -15,6 +16,18 @@ void init_global_game_state() {
     PATH_INFO::init_user_dir("./");
     PATH_INFO::set_standard_filenames();
 
+    if( !assure_dir_exist( FILENAMES["config_dir"] ) ) {
+        assert( !_( "Unable to make config directory. Check permissions." ) );
```

Comment:
Debug messages don't need to be translated. The translation system is probably not even initialized at this point. And strings are probably not extracted (for translation) from the source files in the tests directory anyway.

Finally: asserting on the success of any kind of file system operation seems wrong. They can *always* fail. But this is in test code, so it's probably fine to use assertions to report the failure.